### PR TITLE
[Merged by Bors] - feat(Analysis/PSeries): simp for summable_nat_pow_inv

### PR DESCRIPTION
**Pull request by mo271**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Analysis/PSeries.lean [hunk 1]
```diff
@@ -223,8 +222,7 @@ theorem summable_nat_pow_inv {p : ℕ} :
 if and only if `1 < p`. -/
 theorem summable_one_div_nat_pow {p : ℕ} :
     Summable (fun n => 1 / (n : ℝ) ^ p : ℕ → ℝ) ↔ 1 < p := by
-  -- porting note (#10745): explicitly supplied two simp lemmas
-  simp only [one_div, Real.summable_nat_pow_inv]
+  simp
```
Comment (Parcly-Taxel) [inline Mathlib/Analysis/PSeries.lean:225]: A `simp only` with specified lemmas is more robust than just `simp`. Just remove the porting note.

Comment (mo271) [reply Mathlib/Analysis/PSeries.lean:225]: Sure, I will change it if you think that is better. Of course it is also more verbose...

Comment (mo271) [inline Mathlib/Analysis/PSeries.lean:225]:
```suggestion
  simp only [one_div, Real.summable_nat_pow_inv]
```

Comment (mo271) [reply Mathlib/Analysis/PSeries.lean]: Done

**Review by Ruben-VandeVelde — APPROVED** [review]
maintainer merge

**Pull request closed without merging**